src/ovh-angular-line-diagnostics/translations/Messages_fr_FR.xml

```diff
@@ -182,6 +183,11 @@
 
    <translation id="tools_lineDiagnostics_solution_proposal_step_validationRefused">La demande d'intervention a été refusée par l'assistance technique OVH.</translation>
 
+   <!-- Modal -->
+   <translation id="tools_lineDiagnostics_confirm_modal_title">Confirmation de demande d'intervention</translation>
+   <translation id="tools_lineDiagnostics_confirm_modal_explanation">Afin de finaliser votre demande d'intervention, nous allons réalisé un test technique qui rendra votre <strong>connexion internet indisponible pendant 15 minutes</strong> environ. Merci de prendre toutes les précautions nécessaires à ce cas avant de continuer.</translation>
```

Comment:
Please avoid having `html` in `xml` files

Reply:
I know right, but I'm definitely not gonna split that sentence into meaningless chunks :/

Reply:
True, this is an issue